src/components/remotecontrol/remotecontrol.scss

```
@@ -399,7 +403,8 @@
     }
 
     .nowPlayingPageImage.nowPlayingPageImageAudio {
-        width: 100%;
+        max-width: 100%;
```

Without `width: auto;`, posters don't keep the aspect ratio.
```suggestion
        width: auto;
        max-width: 100%;
```